nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,61 @@
+---
+title: SQL Server transport deployment options
+summary: Describes available SQL Server transport deployment options
+tags:
+- SQL Server
+- Transports
+redirects:
+- nservicebus/sqlserver/multiple-databases
+---
+
+In default configuration SQL Server transport assumes that all tables used for storing messages are located in a single catalog and within a single schema. The configuration can be changed to partition message storage between different schemas and catalogs. The decision can me made on per queue basis (including error and audit queues).
```

Comment:
> In default configuration SQL Server transport assumes that all tables used for storing messages are located in a single catalog and within a single schema.

I think this is better as:
When using the default configuration, SQL Server transport assumes that all tables used for storing messages for endpoints are located in a single catalog and within a single schema. 

>The decision can me made on per queue basis (including error and audit queues).

typo `can me` -> `can be`. 
How about: 
The schemas and catalogs can also be specified at a queue level. For example, the error and the audit queues can be configured to use a different schema and a different database catalog.